# papers with data to mine

There are a lot of published papers with a lot of data and mining data can make new discoveries.

### CCLE 

* [Quantitative Proteomics of the Cancer Cell Line Encyclopedia](https://www.cell.com/cell/fulltext/S0092-8674(19)31385-6)
* [Pan-cancer single cell RNA-seq uncovers recurring programs of cellular heterogeneity](https://www.biorxiv.org/content/10.1101/807552v1.full)
* [Next-generation characterization of the Cancer Cell Line Encyclopedia](https://www.nature.com/articles/s41586-019-1186-3)
Broad has [DepMap](https://depmap.org/portal/)

[depmap bioc package](http://bioconductor.org/packages/release/data/experiment/html/depmap.html)

### TCGA & ICGC

* [The chromatin accessibility landscape of primary human cancers](https://science.sciencemag.org/content/362/6413/eaav1898)
* [Disruption of chromatin folding domains by somatic genomic rearrangements in human cancer](https://www.nature.com/articles/s41588-019-0564-y)

### immune 
* [The cis-Regulatory Atlas of the Mouse Immune System](https://www.cell.com/cell/fulltext/S0092-8674(18)31650-7)

### chromatin interaction
* [3D Genome Browser from Yue Lab](http://promoter.bx.psu.edu/hi-c/)

### chromatin accessibility

* [Global reference mapping and dynamics of human transcription factor footprints](https://www.biorxiv.org/content/10.1101/2020.01.31.927798v1) We map the fine-scale structure of ~1.6 million DHS

* [Index and biological spectrum of accessible DNA elements in the human genome](https://www.biorxiv.org/content/10.1101/822510v2) Download the data from https://www.meuleman.org/project/dhsindex/ Wouter Meuleman is the author of epilogos.

### epigenome annotation

* [Integrative analysis of 10,000 epigenomic maps across 800 samples for regulatory genomics and disease dissection](https://www.biorxiv.org/content/10.1101/810291v2) by Manolis Kellis group: To help elucidate genetic variants underlying complex traits, we develop EpiMap, a compendium of 833 reference epigenomes across 18 uniformly-processed and computationally-completed assays. We define chromatin states, high-resolution enhancers, activity patterns, enhancer modules, upstream regulators, and downstream target gene functions.
